### PR TITLE
Add homework for lesson #42

### PR DESCRIPTION
**Pull request by petroleg**
Please provide code review and comment if necessary:)

### CODE REVIEW

File: djangoProject/user/tasks.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from purchase.models import Purchase
+
+
+@shared_task
+def print_hello_world():
+    print("Hello World from task")
+
+
+@shared_task
+def print_purchases(user_id):
```
Comment (acman) [inline djangoProject/user/tasks.py:13]: use this functions somewhere

**Review by acman — CHANGES_REQUESTED** [review]
Don't see usage of print_purchase

File: djangoProject/user/tasks.py (hunk 1 above)
Comment (petroleg) [reply djangoProject/user/tasks.py:13]: Done

**Review by acman — APPROVED** [review]
GJ!

**Pull request merged**